sdk/identity/src/token_credentials/azure_cli_credentials.rs

```diff
@@ -119,6 +119,11 @@ impl Default for AzureCliCredential {
 }
 
 impl AzureCliCredential {
+    pub fn create() -> azure_core::Result<Self> {
+        // TODO check `az version` to see if it's installed
```

Comment:
I created https://github.com/Azure/azure-sdk-for-rust/issues/1533 to track this TODO.